packages/codemod/src/test/jscodeshift-testUtils.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'jscodeshift/dist/testUtils' {
```

Comment:
naming: `test-utils` (js convention, unless this was intentional for testing?)

Reply:
Agreed re: naming, I think we might need this to match the jscodeshift dist filenames. Yes, it's just for testing so that we get ts-lint validity IIRC. I will look at it separately.